common/data/base/houseBase.ts

```diff
@@ -1,18 +1,28 @@
+import qs from 'qs';
 import { HistogramDataWithoutValues } from 'components/UI/RangeHistogram';
 import { getObjectsTotalCount, parallelRequests, STRAPI_BASE_URL } from '../dataHelpers';
+import { FilterOperator } from '../filterOperator';
 import { getById } from './getById';
 
 export const houseBase = {
     async getObject(id: string): Promise<HouseObject> {
         return getById.getObject(id, '/house');
     },
     async getObjectsPolygonsByRange(from: number, to: number, filterName: string) {
-        const fullUrl = new URL(`${STRAPI_BASE_URL}/house?populate=borders`);
+        const url = `${STRAPI_BASE_URL}/house`;
 
-        fullUrl.searchParams.append(`filters[${filterName}][$gte]`, from.toString());
-        fullUrl.searchParams.append(`filters[${filterName}][$lte]`, to.toString());
+        const query = qs.stringify({
+            filters: {
+                [filterName]: {
+                    [FilterOperator.GreaterThanOrEqual]: from,
```

Comment:
`FilterOperator['=>']` or `FilterOperator['≥']`

or maybe we can try to make such a thing? 😏

Reply:
Then we need copy this symbol every time. i think it's uncomfortable

Reply:
It's about second variant